### ISSUE TITLE
[tvOS] Force-switches to GL when set to Metal

### ISSUE THREAD

**Issue by kerberjg** [libretro/mupen64plus-libretro-nx#512]
First noticed here: https://github.com/libretro/RetroArch/issues/16083, using RetroArch nightly "Jan 6 2024" on tvOS 17.2

When starting an N64 game with this core and RetroArch set to Metal on tvOS, it switches the renderer back to GL.

Not sure if relevant, but RetroArch reports the core build as "Mupen64Plus-Next (2.6-GLES3)", which is weird because the same core from latest stable (1.16.0) works just fine with Metal on M1 Mac, and reports "Mupen64Plus-Next (2.5-Vulkan)".

Are Vulkan builds disabled for tvOS in this core? RetroArch's system information reports both Vulkan and Metal as available.

**Comment by warmenhoven** [comment]
I think I answered this in a different comment to you, but for completeness or in case I only thought I did I'll reply here as well.

Yes, on iOS/tvos to date mupen forces GL because the only renderer available to it was GLideN64 which requires GL.

The angrylion renderer and cxd4 LLE are in now, so that should be an option in the next nightly build for sticking with vulkan. Hopefully the parallel renderer and LLE will go in as well (though the parallel rsp will require a JIT, which in iOS/tvOS 17 is not as easy to get).
